### PR TITLE
Fixes governor unit tests for consistency

### PR DESCRIPTION
**Pull request by isaacm**
Also adds misc. pep8 fixes (on apache 2 licenses)

### CODE REVIEW

File: eom/__init__.py [hunk 1]
```diff
@@ -22,9 +21,7 @@
 
 
 def version():
-    {
     return __version__
-    }
```
Comment (BenjamenMeyer) [inline eom/__init__.py]: My C++ got into me...

File: eom/governor.py [hunk 2]
```diff
@@ -153,6 +152,7 @@ def calc_sleep(project_id, rate):
         now = time.time()
         last_time = now
         count = 1.0
+        new_count = 1.0
```
Comment (BenjamenMeyer) [inline eom/governor.py]: I think ```new_count``` is suppose to be just within the loop, while ```count``` is the final verdict
@TheSriram please verify

Comment (BenjamenMeyer) [inline eom/governor.py]: Looking more closely, I think @isaacm may have hit upon a nice bug in the governor

It looks like ```count``` is the real loop internal variable and should remain ephemeral to the loop, while ```new_count``` should really capture the persistent value that is the culmination of the loop.

The bug that got resolved here was that the final comparison (L186) was against what was in storage, not the value calculated in L176, which was then added to storage (L177).

Perhaps ```new_count``` should really be called ```final_count``` to make this more clear.
@TheSriram  am I reading the loop correctly?

**Comment by BenjamenMeyer** [comment]
:+1:

:money_with_wings:  to @isaacm for the catching the :bug:

**Comment by TheSriram** [comment]
:money_with_wings: lol

**Pull request merged**